config/locales/en_LK.yml

```diff
@@ -52,28 +52,51 @@ en-LK:
     reports_card_subtitle_fbs: "Diabetes patients in %{region_name} with FBS <126 mg/dL or HbA1c <7% at their last visit in the last 3 months"
+    reports_card_subtitle_dm: "Diabetes patients at %{region_name} registered >3 months ago with blood sugar <200 at their last visit in the last 3 months"
+    reports_card_subtitle_dm_fbs: "Diabetes patients at %{region_name} registered >3 months ago with FBS <126 mg/dL or HbA1c <7% at their last visit in the last 3 months"
     summary: "patients with a blood sugar <200 from"
     summary_fbs: "patients with blood sugar controlled from"
   bs_over_200_copy:
-    reports_card_title: "Blood sugar 200-299 or ≥300"
+    reports_card_title: "Blood sugar ≥200"
     reports_card_title_fbs: "Blood sugar uncontrolled"
     reports_card_subtitle: "Diabetes patients in %{region_name} with blood sugar 200-299 or blood sugar ≥300 at their last visit in the last 3 months"
     reports_card_subtitle_fbs: "Diabetes patients in %{region_name} with FBS ≥126mg/dL or HbA1c ≥7% at their last visit in the last 3 months"
     bs_200_to_299:
       title: "Blood Sugar 200-299"
       title_fbs: "Blood Sugar 126-199"
+      title_dm: "Blood Sugar 200-299"
+      title_dm_fbs: "FBS 126-199mg/dL or HbA1c 7%-8.9%"
+      report_card_lower_bar: "Blood Sugar 200-299"
+      report_card_lower_bar_fbs:  "FBS 126-199mg/dL or HbA1c 7%-8.9%"
       numerator: "Patients with RBS/PPBS 200-299, FBS 126-199, or HbA1c 7.0-8.9 at their last visit in the last 3 months"
       numerator_fbs: "Patients with FBS 126-199mg/dL, or HbA1c 7%-8.9% at their last visit in the last 3 months"
+      numerator_dm: "Patients with RBS/PPBS 200-299, FBS 126-199 or HbA1c 7.0-8.9 at their last visit in the last 3 months"
+      numerator_dm_fbs: "Patients with FBS 126-199mg/dL or HbA1c 7%-8.9% at their last visit in the last 3 months"
+      reports_card_subtitle: "Diabetes patients at %{region_name} registered >3 months ago with blood sugar 200-299 at their last visit in the last 3 months"
+      reports_card_subtitle_fbs: "Diabetes patients at %{region_name} registered >3 months ago with FBS 126-199mg/dL or HbA1c 7%-8.9% at their last visit in the last 3 months."
       summary: "patients with a blood sugar 200-299 from"
       summary_fbs: "patients with FBS 126-199mg/dL or HbA1c 7%-8.9% from"
     bs_over_300:
       title: "Blood sugar ≥300"
       title_fbs: "Blood sugar ≥200"
+      title_dm: "Blood sugar ≥300"
+      title_dm_fbs: "FBS ≥200mg/dL or HbA1c ≥9%"
+      report_card_lower_bar: "Blood Sugar ≥300"
+      report_card_lower_bar_fbs:  "FBS ≥200mg/dL or HbA1c ≥9%"
       numerator: "Patients with RBS/PPBS ≥300, FBS ≥200, or HbA1c ≥9.0 at their last visit in the last 3 months"
       numerator_fbs: "Patients with FBS ≥200mg/dL, or HbA1c ≥9% at their last visit in the last 3 months"
+      numerator_dm: "Patients with RBS/PPBS ≥300, FBS ≥200 or HbA1c ≥9.0 at their last visit in the last 3 months"
```

Comment:
All the new keys that are being added in en-LK.yml file, should also be present in en.yml file as well. For any other country other than SL, if the default locale is en, then these will not be found in en.yml file and page would likely break.